frontend/app/package.json

```diff
@@ -56,6 +56,7 @@
     "chartjs-plugin-zoom": "2.2.0",
     "consola": "3.2.3",
     "dayjs": "1.11.13",
+    "dom-to-image": "2.6.0",
```

Comment:
I can't find easy way to do this without 3rd party library.
do you think it's fine? @kelsos 
we can remove it after.